plugins/gateway-backend/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@backstage/plugin-gateway-backend",
+  "version": "1.0.0",
+  "backstage": {
+    "role": "backend-plugin",
+    "pluginId": "gateway",
+    "pluginPackages": [
+      "@backstage/plugin-gateway-backend"
+    ]
+  },
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/backstage/backstage",
+    "directory": "plugins/gateway-backend"
+  },
+  "license": "Apache-2.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "backstage-cli package build",
+    "clean": "backstage-cli package clean",
+    "lint": "backstage-cli package lint",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack",
+    "start": "backstage-cli package start",
+    "test": "backstage-cli package test"
+  },
+  "dependencies": {
+    "@backstage/backend-plugin-api": "workspace:^",
+    "@opentelemetry/api": "^1.9.0",
+    "@opentelemetry/core": "^2.0.0",
+    "express": "^4.17.1",
+    "http-proxy-middleware": "^3.0.3"
```

Comment:
did you explicitly need v3? i guess we can migrate the rest too separately some day

Reply:
I guess same here though, when making the feature PR it's usually a good idea to make the yarn.lock change as small as is humanly possible, and then doing bumps separately. It's just that, when reviewing, it's easy to get stuck up on details like this because like "what if this has a side effect on our next deployment of the catalog so it has broken / changed metrics" etc, maybe unnecessarily but we don't really know.

You don't have to change this now, just mentioning